src/yarpWholeBodySensors.cpp

```diff
@@ -621,6 +910,30 @@ bool yarpWholeBodySensors::openTorqueSensor(const int bp)
     return true;
 }
 
+vector< std::string > yarpWholeBodySensors::getSensorNames(const SensorType st)
```

Comment:
@naveenoid there is a difference between the IDs in the IDList and the sensor names ?